packages/flutter_test/lib/src/controller.dart

```diff
@@ -1683,6 +1736,7 @@ abstract class WidgetController {
       pointer: pointer,
       buttons: buttons,
       frequency: frequency,
+      viewId: viewId ?? 0,
```

Comment:
Can we look this up from the finder if needed?

We shouldn't hardcode `0` as the implicit view ID: embedders should be able to stuff information in the view IDs, like a pointer, so the implicit view's ID might not be 0 on all embedders.

In locations where we do need a fallback view ID, can we look up the `WidgetsBinding.instance.platformDispatcher.implicitView` and throw if that's `null`?